scripts/bootstrap.sh

```diff
@@ -561,6 +571,7 @@ update_devstack()
 ###############################################
 # START CORE EXECUTION
 ###############################################
+set -xv
```

Comment:
Do we need this?

Reply:
This was added for debugging.  I'll remove.